src/compute-client/src/controller/instance.rs

```diff
@@ -1401,18 +1401,22 @@ where
             // to run again.
             true
         } else {
+            let ignore_self_deps = |x: &GlobalId| *x != id;
```

Comment:
Nit: Since this is a predicate, I'd call this `is_self_dep`

Reply:
Done, except s/is_self_dep/not_self_dep/ :)

Reply:
🙈